src/OSPSuite.UI/Views/Importer/ImporterDataView.cs

```diff
@@ -286,10 +291,55 @@ public void SetFilter(string filter)
          useForImportCheckEdit.Checked = true;
       }
 
-      public string SelectedTab { get; set; }
+      private string _selectedTab;
+      public string SelectedTab {
+         get => _selectedTab;
+         set
+         {
+            _selectedTab = value;
+            refreshErrorMessage();
+         }
+      }
       public string GetFilter()
       {
          return DevExpress.Data.Filtering.CriteriaToWhereClauseHelper.GetDataSetWhere(dataViewingGridView.ActiveFilterCriteria);
       }
+
+      private void refreshErrorMessage()
+      {
+
+         if (_lastLoadedDataSets.Contains(SelectedTab) && _lastErrors.Contains(_lastLoadedDataSets[SelectedTab]))
+         {
+            labelControlError.Text = Error.ParseErrorMessage(_lastErrors.ErrorsFor(_lastLoadedDataSets[SelectedTab]).Select(x => x.Message));
+            layoutControlItem2.Visibility = DevExpress.XtraLayout.Utils.LayoutVisibility.Always;
+         }
+         else
+         {
+            labelControlError.Text = "";
+            layoutControlItem2.Visibility = DevExpress.XtraLayout.Utils.LayoutVisibility.Never;
+         }
+      }
+
+      private void refreshErrorMarks()
+      {
+         importerTabControl.TabPages.Each(x =>
+         {
+            if (_lastLoadedDataSets.Contains(x.Text))
```

Comment:
this codes for which image to show also belongs in the presenter 
And hopefully, we won't have those magin number anymore 1 and 0 once we use ApplicationIcons